### DISCUSSION TITLE
New standalone .exe file

### DISCUSSION THREAD

**Discussion by toutjavascript** [toutjavascript/FoooXus-Fooocus-Extender#4] [Announcements]
Here a new standalone .EXE file
It is very heavy.
If you can, run FoooXus with python classical installation
File is too big to be uploaded to github
It will be here https://www.toutjavascript.com/foooxus/ after uploading

<hr /><em>This discussion was created from the release <a href='https://github.com/toutjavascript/FoooXus-Fooocus-Extender/releases/tag/V0.8'>New standalone .exe file</a>.</em>

**Comment by bsdesigner** [comment]
It's not downloading:
![Снимок экрана 2024-02-19 231455](https://github.com/toutjavascript/FoooXus-Fooocus-Extender/assets/36730003/9f4f8f8e-5f1c-4934-9de1-8cfbad2bff71)

**Reply by toutjavascript** [reply]
Hello
The upload is now over
I have updated the release.
Can you test it please?

**Reply by bsdesigner** [reply]
Everything seems to be working: 
![Снимок экрана 2024-02-20 131006](https://github.com/toutjavascript/FoooXus-Fooocus-Extender/assets/36730003/6a4c43fa-5225-4aa3-82d6-686c07476c27)
![Снимок экрана 2024-02-20 131216](https://github.com/toutjavascript/FoooXus-Fooocus-Extender/assets/36730003/af3b2e05-17eb-4742-a110-7dd98d40171e)

**Reply by bsdesigner** [reply]
But it takes a very long time to render:
![Снимок экрана 2024-02-20 131055](https://github.com/toutjavascript/FoooXus-Fooocus-Extender/assets/36730003/8f14ca78-236c-4ab4-803b-ded67c0d5273)

**Reply by bsdesigner** [reply]
I changed the base model to Turbo, but didn't realise where to change the other parameters for it to work correctly:
![Снимок экрана 2024-02-20 131448](https://github.com/toutjavascript/FoooXus-Fooocus-Extender/assets/36730003/e45fa471-a8b5-419a-9f63-0d9115203cfc)

**Comment by toutjavascript** [comment]
Thanx
Happy the .exe is working

You could change the parameters in config.json
I will add full informations on the file https://github.com/toutjavascript/FoooXus-Fooocus-Extender/blob/main/configpreset.md

Styles are generating quite fast on my hardware config (with .exe or simple native python)
About 20 seconds (ryzen 3950X and RTX370)
The first generation launch is longer because it must load model
The next generations are very faster

**Reply by bsdesigner** [reply]
I have an RTX 2070 (8Gb) on my computer, unfortunately...

**Reply by bsdesigner** [reply]
In the file "config.json", there is no possibility to set optimal parameters for the model "dreamshaperXL_v21TurboDPMSDE". And with such parameters, the rendering would be much faster....
![Снимок экрана 2024-02-20 140904](https://github.com/toutjavascript/FoooXus-Fooocus-Extender/assets/36730003/81b98650-52c1-4c49-b7c4-ade1628e51f7)

**Reply by bsdesigner** [reply]
Anyway, I would like to thank you for the important and wonderful work you have done. Thank You Very Much!

**Reply by bsdesigner** [reply]
And a special huge thank you, for Fooocus Log Viewer!!!

**Reply by toutjavascript** [reply]
Yes, it is possible to change that parameters
Please wait till tomorrow I write the help doc

Thans for all

**Reply by toutjavascript** [reply]
Hello @bsdesigner 
I have updated the doc file about preset

**Reply by bsdesigner** [reply]
Thank You!
